websites/N/Necord/presence.ts

```diff
@@ -0,0 +1,131 @@
+enum SmallImageKeys {
+	Idle = "https://i.imgur.com/6A9pxAR.png",
+	Search = "https://i.imgur.com/T9NzckB.png",
+	Read = "https://i.imgur.com/ULFqucm.png",
+}
+
+const presence = new Presence({
+		clientId: "1081479845940314114",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	IDLE_TIMEOUT = 10 * 60 * 1000,
+	settings = [
+		"search",
+		"docs-page",
+		"docs-page-content",
+		"docs-page-sidebar",
+		"idling",
+	];
+
+let sidebar: string = null,
+	lastActivity: number = Date.now();
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/LY804vK.png",
+			startTimestamp: browsingTimestamp,
+		},
+		{ pathname, search } = window.location,
```

Comment:
document is prefered over window. Also add href here.
```suggestion
		{ href, pathname, search } = document.location,
```